README.md

```diff
@@ -69,6 +69,20 @@ The command line interface (CLI) allows users to submit their project package de
    phylum analyze <package-lock-file.ext>
    ```
 
+## Uninstalling the Phylum CLI
+
+The Phylum CLI can be uninstalled using the `phylum_uninstall.sh` script, which
+should be in your `$PATH` and at `~/.phylum/phylum_uninstall.sh` if you followed
```

Comment:
Reminder to ensure this section has file names that match whatever the ultimate name is: `uninstall-phylum.sh` is the current suggestion.

Another reminder is that an odd quirk of the way the docs are set up in this repo...and published to a README.com site at <https://docs.phylum.io/docs/welcome> ...is that the `README.md` file should be manually kept in sync with the `docs/quickstart.md` file. Looks like that is the case now, but just repeating here instead of also adding a comment to that file.